docs/Testing-Releases.md

```diff
@@ -32,6 +32,7 @@ Here are the tags for Docker images that should be noted:
 * healthplatformregistry.azurecr.io/${version}_fhir-server:**master** - The latest code that has been merged to the main branch and passed all tests.
 * healthplatformregistry.azurecr.io/${version}_fhir-server:**release** - The latest release, this is the tag we will focus on.
 * healthplatformregistry.azurecr.io/${version}_fhir-server:**build-[release number]** - Allows you to pin or access a specific release.
+* mcr.microsoft.com/healthcareapis/${version}-fhir-server:**latest** - If the image has issues with the above tags, consider this.
```

Comment:
We probably want to change the image this document assumes the user will want to be this one since it is the official release now.

Reply:
Good point! Reworded and Re-ordered these images so that the focused one appears first. Let me know if this looks ok?